uw-frame-components/portal/timeout/services.js

```diff
@@ -11,8 +11,9 @@ define(['angular', 'jquery'], function(angular, $) {
     * getTimeout() - Gets the shib session, but returns just the timeout
                      and the time of that timeout.
     **/
-    .factory('PortalShibbolethService', ['$http', 'miscService', 'SERVICE_LOC',
-        function($http, miscService, SERVICE_LOC) {
+    .factory('portalShibbolethService', ['$http', '$log',
```

Comment:
Maybe. 

I think we'd be able to take seriously a requirement for Shibboleth to be optional if and when a non-Shib-using adopter arrives. In practice the Shib relationship is about headers, headers that are relatively easily placed by a variety of authentication environment solutions, including CAS.

It may be that all `uPortal-home` adopters in the next 5 years are Shibboleth adopters. Shibboleth is awfully common among our target adopters.

As an actionable next step: I think clarifying our documentation about this philosophic stance would be a fine thing. The short version is that I think there'd be tremendous willingness to factor out weird Wisconsin-isms at the moment where they impede an adopter.